src/Sulu/Bundle/ContentBundle/Search/Metadata/StructureDriver.php

```diff
@@ -153,6 +153,11 @@ public function loadMetadataForClass(\ReflectionClass $class)
             'field' => $this->factory->createMetadataProperty('webspaceKey'),
         ));
 
+        $indexMeta->addFieldMapping('state', array(
+            'type' => 'string',
+            'field' => $this->factory->createMetadataExpression('object.nodeState == 1 ? "test" : "published"'),
```

Comment:
This is a bit shit, but I don't think we should be asking the developer to do `webspace_key:sulu_io state:1`, and it certainly doesn't make sense from the UI POV.

We should migrate to using strings instead of numbers for all things enumerable.